liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -185,28 +204,39 @@ private boolean mavenWarPluginExists(MavenProject proj) {
         }
         return false;
     }
-    
+
     private boolean matches(Artifact artifact, ArtifactItem assemblyArtifact) {
         return artifact.getGroupId().equals(assemblyArtifact.getGroupId())
                 && artifact.getArtifactId().equals(assemblyArtifact.getArtifactId())
                 && artifact.getType().equals(assemblyArtifact.getType());
     }
-    
-    private boolean isSupportedType(String type) {
+
+    private boolean isSupportedType(String type) throws Exception {
```

Comment:
I don't think throwing an exception here is right, based on what the calling method does. I think you should just log a warning with the information below and then the calling method will handle accordingly based on the return of false.